gci/management/commands/filter_gci_non_issue_tasks.py

```diff
@@ -0,0 +1,40 @@
+import os.path
+
+from ruamel.yaml import YAML
+from django.core.management.base import BaseCommand
+
+from gci.gitorg import get_issue
+
+
+class Command(BaseCommand):
+    help = 'Extract the tasks which dont have an issue associated with them'
+
+    def add_arguments(self, parser):
+        parser.add_argument('output_dir', type=str,
+                            help='directory of tasks.yaml file')
+
+    def handle(self, *args, **options):
+        output_dir = options.get('output_dir')
+
+        non_issue_tasks = {}
+
+        yaml = YAML()
+
+        try:
+            with open(os.path.join(output_dir, 'tasks.yaml')) as f:
+                tasks = yaml.load(f)
+
+                for task in tasks:
+                    issue = get_issue(tasks[task]['external_url'])
+                    print('external issue for task is {}'.format(issue))
+                    if not issue:
+                        print('task {} is a non-issue task'.format(task))
+                        non_issue_tasks[task] = tasks[task]
+                    print('task {} has a valid issue link'.format(task))
```

Comment:
print out the URL here , so you can easily check them ... currently it is producing rubbish.